packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -28,5 +28,10 @@
     vertical-align: middle;
   }
 
-
+  // People Picker specific override to prevent truncation of Persona name
+  .ms-Persona {
```

Comment:
Can you remove the `.ms-Persona` wrapper and comment? Or replace it with another class name?